css.css

```diff
@@ -1,104 +1,204 @@
 body {
 	font-family: helvetica, arial, sans-serif;
 	margin: 0;
+    background: linear-gradient(318deg, #0005ff, #c3d014, #e42f2f, #0005ff, #c3d014, #e42f2f);
+    background-size: 800% 800%;
+
+    -webkit-animation: AnimationName 30s ease infinite;
+    -moz-animation: AnimationName 30s ease infinite;
+    animation: AnimationName 30s ease infinite;
+}
+
+@-webkit-keyframes AnimationName {
+    0%{background-position:0% 11%}
+    50%{background-position:100% 90%}
+    100%{background-position:0% 11%}
+}
+@-moz-keyframes AnimationName {
+    0%{background-position:0% 11%}
+    50%{background-position:100% 90%}
+    100%{background-position:0% 11%}
+}
+@keyframes AnimationName {
+    0%{background-position:0% 11%}
+    50%{background-position:100% 90%}
+    100%{background-position:0% 11%}
 }
 
-h1 {
-	font-style: italic;
+h1, p, h2, em {
 	color: #ffffff;
 }
 
+.header h1 {
```

Comment:
Added the needed div classes to specify and segment the information on the webpages to be able to organise and control the information presented as desired